packages/@expo/cli/src/start/index.ts

```diff
@@ -77,6 +78,7 @@ export const expoStart: Command = async (argv) => {
 
     --dev-client                           Experimental: Starts the bundler for use with the expo-development-client
     --force-manifest-type <manifest-type>  Override auto detection of manifest type
+    --private-key-path <path>              Path to private key for code signing. Default: "private-key.pem" in the same directory as the certificate specified by the expo-updates configuration in app.json.
```

Comment:
The start command does a lot of stuff, maybe `signing-key` would be a more useful name? This would be akin to `serve` CLI which uses `--ssl-key`.
```suggestion
    --signing-key <path>              Path to private key for code signing. Default: "private-key.pem" in the same directory as the certificate specified by the expo-updates configuration in app.json.
```

Reply:
If we rename this flag, we probably should rename the default file too to keep the current consistency in this PR. E.g. if the flag is `--code-signing-key` then the file should be `code-signing-key.pem` unless we intend for the filename to differ.